packages/shared/src/hooks/post/useComments.ts

```diff
@@ -0,0 +1,64 @@
+import { useCallback, useMemo, useState } from 'react';
+import { Comment } from '../../graphql/comments';
+import { CommentMarkdownInputProps } from '../../components/fields/MarkdownInput/CommentMarkdownInput';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { isNullOrUndefined } from '../../lib/func';
+import { useAnalyticsContext } from '../../contexts/AnalyticsContext';
+import { postAnalyticsEvent } from '../../lib/feed';
+import { AnalyticsEvent, Origin } from '../../lib/analytics';
+import { Post } from '../../graphql/posts';
+
+type ReplyTo = [Comment, string, boolean?];
+
+const initialState: ReplyTo = [null, null, false];
```

Comment:
What are these values, not super clear from a review point of view.

Reply:
The initial state of the ReplyTo object. We have a state we use in the component and this is the initial value. We want to be able to return to this initial state at some point so instead of explicitly doing it twice, we are keeping it in a variable.

Reply:
Ah, right. Let me type it up to make it explicit.